Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py

```diff
@@ -289,30 +281,21 @@ def send_new_mail_request(incident_id, email_subject, subject_include_incident_i
     else:
         subject_with_id = email_subject
 
+    mail_content = {
+        "to": email_to,
+        "subject": subject_with_id,
+        "cc": email_cc,
+        "bcc": email_bcc,
+        "htmlBody": email_html_body,
+        "body": email_body,
+        "attachIDs": ",".join(entry_id_list),
+        "replyTo": service_mail,
+        "bodyType": body_type,
```

Comment:
Nitpick. Not a must. :-)
```suggestion
        "htmlBody": email_html_body,
        "bodyType": body_type,
        "body": email_body,
        "attachIDs": ",".join(entry_id_list),
        "replyTo": service_mail,
```